include/tlsuv/src_t.h

```diff
@@ -13,23 +13,20 @@
 // limitations under the License.
 
 /**
- * @file um_http_src_t.h
- * @brief header file for um_http_src_t type
+ * @file ttp_src_t.h
+ * @brief header file for tlsuv_src_t type
```

Comment:
the typedef now removed, so should this be `struct tlsuv_src_s`?

Reply:
the type is still there, the warning was because it `typdef`ed in both places